.github/workflows/tv-os-build-test.yml

```diff
@@ -27,6 +27,13 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v4
+
+      - name: Select Xcode 16.1
+        run: sudo xcode-select -s /Applications/Xcode_16.1.app
+
+      - name: Check Xcode version
+        run: xcodebuild -version
```

Comment:
Is this promised that the Xcode will be present in the environment? 

Usually we use the `setup-xcode` workflow, like [here](https://github.com/software-mansion/react-native-screens/blob/546318086b7035b7ec99663ff3aaff3065b186af/.github/workflows/ios-e2e-test-fabric.yml#L21-L24)

Reply:
https://github.com/software-mansion/react-native-screens/pull/3465/commits/cbb7e045ae8717d8d9569406eb05a1f8c38eee0f